packages/legend-extension-external-language-morphir/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './MorphirGenerationPlugin';
```

Comment:
Please add the copyright header. You can do that automatically with `yarn fix:copyright`

Reply:
Thanks! Fixed.